.travis.yml

```diff
@@ -0,0 +1,32 @@
+language: node_js
+sudo: required
+env:
+  global:
+    - firebase_bucket=gs://dracco1993-tba-dev.appspot.com
```

Comment:
We'll need to change this to point to prod

Reply:
Done in 05e95196286d5c0cc431e8148796bf3e8f04f920